### ISSUE TITLE
Game stuttering when receiving messages

### ISSUE THREAD

**Issue by darkcrusade49** [funkydude/BasicChatMods#93]
## Describe the problem
Game freezes/stutters for about 5-10 seconds whenever i get messaged in game (in game PMs or Bnet PMs) or switch chat tabs

### What steps will reproduce the problem?

1.  Enable the addon
2.  Receive an in game PM or Bnet message
3.  Game stutters everytime and the game works fine if the addon is disabled

### Consider attaching a screenshot below to help describe your issue (Attach directly, do not link to other websites)
N/A

### What version of the addon are you using? (Stating 'latest' is not useful)
10.2.7  

### Do you have an error log of what happened?
N/A

**Comment by funkydude** [comment]
Cannot reproduce the problem, does it happen with only BCM enabled?

**Comment by darkcrusade49** [comment]
> Cannot reproduce the problem, does it happen with only BCM enabled?

yeah, everytime i disable this addon in game messages do not lag me but with the addon enabled it does lag me

**Comment by funkydude** [comment]
Thanks but that is not what I asked. I asked you to disable all your other addons except for BCM and see if the issue still occurs.

**Comment by darkcrusade49** [comment]
after turning all addons off except for BCM, the issue does not persist but it does with another addon which might be on their end

**Comment by funkydude** [comment]
Which specific addon?

**Comment by darkcrusade49** [comment]
I am using elvui and i am running version 13.74. I also ran their debug mode and its showing me this
![WoWScrnShot_081524_181110](https://github.com/user-attachments/assets/7dc1992a-a4e1-4ceb-aec3-fd18501d4eed)

**Comment by funkydude** [comment]
ElvUI has chat mods built in, so it's not a surprise that both are conflicting with each other, it's like trying to use 2 chat mods at the same time.

**Comment by darkcrusade49** [comment]
I never knew that. I've been using BCM and ElvUI together for many years and never had this issue until patch 11.0 came out.

**Comment by funkydude** [comment]
That mostly sounds like luck, but you can ask the ElvUI folk to look into it and see what they say, or if they want me to change something to be more compatible.

**Comment by darkcrusade49** [comment]
I'm asking them right now in their discord to see what can be done. Thank you.
